src/Http/Headers/src/ContentDispositionHeaderValue.cs

```diff
@@ -30,6 +30,7 @@ public class ContentDispositionHeaderValue
         private const string ModificationDateString = "modification-date";
         private const string ReadDateString = "read-date";
         private const string SizeString = "size";
+        private const int MaxStackSize = 256;
```

Comment:
Nit: I would rename this to something like MaxStackAllocSizeBytes for clarity.

Reply:
Applying this suggestion myself since that's the only minor issue. Will merge after clean CI.